archiver/main.go

```diff
@@ -40,8 +40,10 @@ const (
 
 const (
 	campaignsFilename          = "campaigns/knowbe4_campaigns.json"
-	groupsFilename             = "groups/knowbe4_groups.json"
+	groupsPath                 = "groups/"
+	groupsFilename             = groupsPath + "knowbe4_groups.json"
 	phishingTestsFilename      = "campaigns/pst/knowbe4_security_tests.json"
+	riskScoreHistoryFilename   = "risk_score_history.json"
```

Comment:
Is there any reason to not do something like what you did with `groupsFilename` here?

Reply:
I see the inconsistency, thanks. Kate just requested a change to this, so it's irrelevant now.